### ISSUE TITLE
Description of EXPTL category

### ISSUE THREAD

**Issue by publcif** [COMCIFS/cif_core#476]
Currently the EXPTL category is described as:

The CATEGORY of data items used to specify the experimental work
prior to diffraction measurements. These include crystallization
crystal measurements and absorption-correction techniques used.

But with the addition of _exptl.method this seems to need updating (perhaps at least remove 'prior to diffraction measurements')?

**Comment by jamesrhester** [comment]
Yes, this description definitely needs to be updated, in a way that captures all of the sub-categories of EXPTL as well as EXPTL itself. It's almost like it contains everything about the experiment and sample except for diffraction-specific items.

How about:
"The category of data items used to specify aspects of the experiment not directly associated with diffraction measurements."
